db/migrate/20220221090435_create_studios.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStudios < ActiveRecord::Migration[6.1]
+  def change
+    create_table :studios do |t|
+      t.string :name
+      t.text :address
```

Comment:
I think it makes sense to store the location lng/lat as that is more accurate for maps. We can store the address, too, but lng/lat will guarantee the location. 

There is more about it in a future lecture - https://kitt.lewagon.com/camps/795/lectures/05-Rails%2F08-Airbnb-Geocoder#source

Reply:
I was thinking we would use the service we used a few weeks ago to get coordinates based on the address. Sure, we could run that during the studio creation and store the coordinates.
We also need the address, because when you register an address online, you're never asked for lng/lat coordinates. Also, the rentee will need the address because it has extra information (like apartment number, name etc...)

Reply:
That can also a feature that can be handled in an another branch.

Reply:
The UI will require an address from the user, definitely, but when we want to display the map, the API will want lng/lat, not address. So rather than geocoding the address every time we display a map, we can geocode once, store, the lng/lat, then use that to communicate with the API, and the address will just be for the UI.